install/kubernetes/values.yaml

```diff
@@ -178,7 +178,23 @@ export:
   filenames:
   - tetragon.log
 
+  # stdout specific exporter settings
   stdout:
+    # should the manifest command be overriden?
+    enabledCommand: true
+
+    # should the manifest args be overridden?
+    enabledArgs: true
+
+    # specific manifest command to use
+    commandOverride: [ ]
+
+    # specific manifest args to use
+    argsOverride: [ ]
+
+    # Extra volume mounts to add to stdout export pod
+    extraVolumeMounts: [ ]
```

Comment:
extraVolumeMounts, commandOverride, and argsOverride are now no longer used in `export`, they should be removed.

Reply:
Good point. I removed them from the values.yaml